components/automate-gateway/api/iam/v2/common/tokens.proto

```diff
@@ -4,11 +4,19 @@ package chef.automate.api.iam.v2;
 option go_package = "github.com/chef/automate/components/automate-gateway/api/iam/v2/common";
 
 message Token {
+    // Unique, optionally user-specified ID. Cannot be changed.
     string id = 1;
+    // User-specified name.
```

Comment:
Here you are using "user-specified" frequently but the policy endpoints do not. Though my personal preference is to omit that phrase (because seems like it should be on every field), but I would say it is more important to be consistent, one way or the other through all endpoints.

Reply:
Yep @tylercloke and I are on it